superset-frontend/src/explore/components/controls/SelectAsyncControl/index.jsx

```diff
@@ -68,7 +68,7 @@ const SelectAsyncControl = props => {
   };
 
   return (
-    <div>
+    <div data-test="SelectAsyncControl">
```

Comment:
Small nit again about the naming convention being with hyphens or just one word lowercase

Reply:
Yes, normally I do this, but in the case of this test (And others similar to this), I thought it would be clearer to keep the `test-id` with the same name as the component.
This will make it clearer at the time of reading and will also help with other tests to identify which html of this component. 

When the HTML is very large it is very hard to find the limits between the components.

Reply:
I'm on the fence here... but I'm OK with it as long as the data-test attr is at the outer element of the component like it is here, and represents the entirety of the component. Little pieces within the component, like `subscribe-button` or whatever, should folow the convention.